### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by daniel-brenot**
Updated dependencies to newest versions. Tests all still pass, but the exception is one test that seems to be failing due to your ipfs link no longer working. You may want the test to just reference a file pushed to the repo if it isn't too large, as links will usually expire at some point.

### CODE REVIEW

**Comment by sameer** [comment]
Ah I'm not running my IPFS node anymore, will see if i can find the original file for `QmSafVXLeEYiSmofSxKcWCAHfRz9EwHAkS1tYDRPtxt7qz`.

**Review by sameer — APPROVED** [review]
LGTM

**Comment by daniel-brenot** [comment]
Only thing preventing this is the test. Do you want me to just remove this test or do you have the file?

**Comment by sameer** [comment]
I unfortunately could not find the file. I tried building it again but the site I originally used has strict rate limits. I think it's fine to put a `#[ignore]` on the test for now

File: src/tokens.rs [hunk 1]
```diff
@@ -308,6 +309,25 @@ pub struct Weather {
     pub condition: Option<Condition>,
 }
 
+impl From<Weather> for String {
```
Comment (sameer) [inline src/tokens.rs:312]: Any reason not to implement [Display](https://doc.rust-lang.org/std/fmt/trait.Display.html)?

Comment (daniel-brenot) [reply src/tokens.rs:312]: Nope. I can implement display as well, the intent was just to allow for writing fields of the metar. I'd eventually like to look at making this library work bidirectionally for reading and writing metars.

Comment (sameer) [reply src/tokens.rs:312]: I think that will create a great opportunity for fuzzing. We can assert that `parse(metar_str) = parse(parse(metar_str).to_string())`

(Ignoring the error case)

**Pull request merged**